app/javascript/src/Policies/util.jsx

```diff
@@ -35,10 +36,45 @@ function generateGuid (): string {
   return s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4() + '-' + s4() + s4() + s4()
 }
 
+function parsePolicies (registry: RawRegistry): Array<RegistryPolicy> {
+  let policies: Array<RegistryPolicy> = []
+  for (let key in registry) {
+    registry[key].forEach(policy => policies.push(parsePolicy(key, policy)))
+  }
+  return policies
+}
+
+function parsePolicy (key: string, policy: RawPolicy): RegistryPolicy {
+  return Object.assign({}, policy,
+    {
+      name: key,
+      humanName: policy.name,
+      data: {}
+    })
+}
+
+const toJson = (val: Object): string => JSON.stringify(val, null, 2)
```

Comment:
Are `toJson` and `fromJson` really necessary?

Also I think their names are upside down, for instance `toJson` gets a JSON and returns a string.

Reply:
Well, would be quite verbose to name it `toJsonString` and `fromJsonString`. And about necessity, calling the `stringify` method with those values all the time, better to encapsulated in a fn, and to keep consistency, make the other one as well.

Reply:
It's not necessary name them like so, simply swap the implementations. _toJson_ sounds like you're getting a JSON, so first thing I'd try is `const json = toJson("{}")`. And vice versa.

On the other hand I think those names would not be verbose at all, they have rather reasonable length for a function in fact.

If you still don't agree, at least changing `val -> json` and `json -> jsonString` would be more accurate.